frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -294,6 +294,19 @@ function ReaderHighlight:genPanelZoomMenu()
             end,
             separator = true,
         },
+        {
+            text = _("Fallback text selection"),
+            checked_func = function()
+                return self.fallback_text_selection
```

Comment:
Fallback **to** text selection - probably. @Frenzie?
(In the text, the variable, the function name.)

Reply:
That'd be "fall back to…"